pkgs/by-name/sp/spl/package.nix

```
@@ -0,0 +1,29 @@
+{
+  pkgs,
+}:
+let
+  version = "0.3.1";
+  src = pkgs.fetchgit {
+    url = "https://git.tudbut.de/tudbut/spl";
+    rev = "v${version}";
+    hash = "sha256-Bj3voUwyqUWg1f86l/Vob+kb5jJUZmf0ZG9IxyYjgh0=";
+  };
+in
+pkgs.rustPlatform.buildRustPackage {
+  pname = "spl";
+  version = version;
+
+  src = src;
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
+  };
+
+  meta = {
+    description = "Stack Programming Language";
```

```suggestion
    description = "Simple, concise, concatenative scripting language";
```
The description should describe the program. Read the [guidelines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#conventions) for more information.